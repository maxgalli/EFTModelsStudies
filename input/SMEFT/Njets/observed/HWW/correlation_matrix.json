{
    "r_Njets_0": {
        "r_Njets_0": 1.0,
        "r_Njets_1": 0.10344547599576216,
        "r_Njets_2": 0.19809058801987472,
        "r_Njets_3": 0.04093970003155676,
        "r_Njets_4": 0.09676302350999631
    },
    "r_Njets_1": {
        "r_Njets_0": 0.10344547599576216,
        "r_Njets_1": 1.0,
        "r_Njets_2": -0.138094816220276,
        "r_Njets_3": 0.15016808351731153,
        "r_Njets_4": 0.05431520305241881
    },
    "r_Njets_2": {
        "r_Njets_0": 0.19809058801987472,
        "r_Njets_1": -0.138094816220276,
        "r_Njets_2": 1.0,
        "r_Njets_3": -0.3640498206968229,
        "r_Njets_4": 0.2094305862241772
    },
    "r_Njets_3": {
        "r_Njets_0": 0.04093970003155676,
        "r_Njets_1": 0.15016808351731153,
        "r_Njets_2": -0.3640498206968229,
        "r_Njets_3": 1.0,
        "r_Njets_4": -0.2668831804553244
    },
    "r_Njets_4": {
        "r_Njets_0": 0.09676302350999631,
        "r_Njets_1": 0.05431520305241881,
        "r_Njets_2": 0.2094305862241772,
        "r_Njets_3": -0.2668831804553244,
        "r_Njets_4": 1.0
    }
}