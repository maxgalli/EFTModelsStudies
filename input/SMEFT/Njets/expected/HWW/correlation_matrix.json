{
    "r_Njets_0": {
        "r_Njets_0": 1.0,
        "r_Njets_1": -0.019794963243805884,
        "r_Njets_2": 0.022888258075205368,
        "r_Njets_3": -0.00569712824479302,
        "r_Njets_4": 0.00211869292111622
    },
    "r_Njets_1": {
        "r_Njets_0": -0.019794963243805884,
        "r_Njets_1": 1.0,
        "r_Njets_2": -0.26527316970378223,
        "r_Njets_3": 0.0850886057854691,
        "r_Njets_4": 0.00010938705553338857
    },
    "r_Njets_2": {
        "r_Njets_0": 0.022888258075205368,
        "r_Njets_1": -0.26527316970378223,
        "r_Njets_2": 1.0,
        "r_Njets_3": -0.3699794242963237,
        "r_Njets_4": 0.09963091422800871
    },
    "r_Njets_3": {
        "r_Njets_0": -0.00569712824479302,
        "r_Njets_1": 0.0850886057854691,
        "r_Njets_2": -0.3699794242963237,
        "r_Njets_3": 1.0,
        "r_Njets_4": -0.31010017241776794
    },
    "r_Njets_4": {
        "r_Njets_0": 0.00211869292111622,
        "r_Njets_1": 0.00010938705553338857,
        "r_Njets_2": 0.09963091422800871,
        "r_Njets_3": -0.31010017241776794,
        "r_Njets_4": 1.0
    }
}